locales/en-US/server.ftl

```diff
@@ -18,6 +18,17 @@ screenshotsLogo =
 bannerSignIn = <a>Sign in or sign up</a> to access your shots across devices and save your favorites forever.
 bannerUpsell = {gScreenshotsDescription} <a>Get Firefox now</a>
 
+# Text used in Firefox Account onboarding promo shown below
+# Sign in button in header
+onboardingPromoTitle = What’s new with Firefox Screenshots?
+onboardingPromoMessage = Now, sign in to Screenshots with a Firefox Account and do more:
+onboardingPromoMessageListItem1 = Access your library on all of your devices
+onboardingPromoMessageListItem2 = Store your favorite shots forever
+onboardingPromoDismissButton = Dismiss
+    .title = Dismiss
+onboardingPromoSigninButton = Sign in
```

Comment:
Not a blocker for localization, but should this use Title Case ("Sign In") or Sentence case ("Sign in")? From the rest of the file, title case seems to be used in buttons.

Reply:
Thanks for catching it, updated to use title case